bin/p2-fetch-manifest/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/manifest"
+	"github.com/square/p2/pkg/store/consul"
+	"github.com/square/p2/pkg/store/consul/consulutil"
+	"github.com/square/p2/pkg/store/consul/flags"
+	"github.com/square/p2/pkg/version"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+)
+
+const (
+	cmdFetchText = "fetch"
+)
+
+var (
+	cmdFetch     = kingpin.Command(cmdFetchText, "Fetch the manifest for a given pod from the datastore and write it to a file on the disk. Combines well with p2-launch.")
```

Comment:
can I say I feel weird asking for a command when there's only one possible choice? Are we leaving room for future expansion, if there will be future commands? If not, what of all the other commands in this directory that ignore the command? p2-schedule, p2-watch, p2-wipe-reality ...